### DISCUSSION TITLE
List of startups: Keep it or lose it?

### DISCUSSION THREAD

**Discussion by turoczy** [portland-startups/docs#21] [Ideas]
Clearly, the most maintenance will center around the list of Portland startups. 

I like it because it gives people a better concept of all of the companies being built here, but it might be overcomplicating what is an otherwise fairly straightforward and easy to maintain list of resources. 

What do you think? Keep the list of startups or remove it?

**Comment by theindiehost** [comment]
Managing it might be a pain in the ***, but I say keep it.

**Comment by tylerwalts** [comment]
Agreed, I like seeing it.

However, the Defunct ones are distracting even though they are nostalgic... what about adding a link off to another page, like `defunct.rst`, `acquired.rst` or `archived.rst`?  This may need some guidance about the lifecycle of an entry and when it's appropriate to sunset one.

**Reply by turoczy** [reply]
Good point. Maybe we should just cordon off the "Active" ones from the defunct/acquired/archived ones to clean it up. 

I'm hesitant to create additional files currently, but recognize that may become a necessity as this continues to grow.

**Comment by turoczy** [comment]
Took a minute to separate "acquired" companies from the primary listing. Will do the same with "defunct" companies when I get a second.

**Comment by turoczy** [comment]
Changed "Acquired" heading to "Exited" since there are a handful of companies that have had different forms of exit, beyond acquisition. e.g., merger, IPO

**Comment by tylerwalts** [comment]
Looks great!!!

**Comment by turoczy** [comment]
Appreciate the feedback! Going to close this one, but I have another question https://github.com/portland-startups/docs/discussions/80
